entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/query/OCondition.kt

```diff
@@ -117,11 +117,21 @@ class OEdgeExistsCondition(
     }
 }
 
+class OSingleVertexCondition(
+    val rid:String
+):OCondition {
+    override fun sql(builder: StringBuilder) {
```

Comment:
Better to leave a line after 
```suggestion
    
    override fun sql(builder: StringBuilder) {
```